dev/airflow-github

```diff
@@ -107,6 +109,71 @@ def is_pr(issue: Issue) -> bool:
     return "apache/airflow/pull/" in issue.html_url
 
 
+def files_touched(repo, commit_id: str) -> List[str]:
+    real_commit = repo.commit(commit_id)
+    return real_commit.stats.files.keys()
+
+
+def is_core_commit(files: List[str]) -> bool:
+    # We list out things that _aren't_ core files,
+    # and we want to know if the commit changes anything
+    # outside of these designated-non-core files
+
+    non_core_files = [
```

Comment:
Very true. Since this is excluding commits that only have changes in these files, new or moved things will pop into the list to be categorized, so at least it'll fail in a safe direction 👍

Reply:
I also think it's okay that the list gets out of date - it can be updated by the release manager when they are categorizing stuff as they'll be looking at the PRs then anyways.